src/Commands/ExportRequestDocsCommand.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Rakutentech\LaravelRequestDocs\Commands;
+
+use Illuminate\Console\Command;
+use Rakutentech\LaravelRequestDocs\LaravelRequestDocs;
+use Rakutentech\LaravelRequestDocs\LaravelRequestDocsToOpenApi;
```

Comment:
use Exception;
use ErrorException;

Reply:
you are mentioning using **use namespace** insist of direct class path call got it